### ISSUE TITLE
model.fit() note working

### ISSUE THREAD

**Issue by dansileshi** [fchollet/keras#4041]
Hello,
I am using keras with tensorflow 
('keras: ', '1.1.0')
('Tensorflow: ', '0.10.0')

I am able to build a ConvNet model using 
model.compile(loss='categorical_crossentropy',  optimizer='sgd',  metrics=['accuracy'])
model.summary()
I think my model defnition doesn't have problem. 
BUT when I tried to train the model with 
model.fit(X_train, Y_train, batch_size=batch_size,  nb_epoch=nb_epoch,verbose=1,       validation_split=0.2)
the ipython notebook is crashing. any help?

**Comment by kgrm** [comment]
This problem is impossible to diagnose with the information you've provided. Please post

* The shape and datatype of your `X_train` and `Y_train` tensors
* The output of `model.summary()`
* The code you're trying to run
* The error(s) it outputs

**Comment by kumarUjjawal** [comment]
![screen shot 2018-04-20 at 11 38 35 am](https://user-images.githubusercontent.com/10569915/39033406-7852cd72-448f-11e8-97ab-17cb021d2b6f.png)
![screen shot 2018-04-20 at 11 38 21 am](https://user-images.githubusercontent.com/10569915/39033407-788b193e-448f-11e8-8f88-442b38d1f6a4.png)
